### PR TITLE
Fixes #11781 - added support for dhcp_server option in dhcp config

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -172,6 +172,8 @@
 #
 # $dhcp_nameservers::           DHCP nameservers
 #
+# $dns_server::                 Address of DHCP server to manage
```
Comment (domcleal) [inline manifests/init.pp]: dhcp, not dns

File: manifests/params.pp [hunk 2]
```diff
@@ -166,6 +166,7 @@
   # if you need to. You can make this a comma-separated string too - it
   # will be split into an array
   $dhcp_nameservers = 'default'
+  $dhcp_server            = '127.0.0.1'
```
Comment (domcleal) [inline manifests/params.pp]: whitespace/indentation looks off

**Comment by witlessbird** [comment]
@domcleal: fixed

**Comment by domcleal** [comment]
:+1:

**Comment by mmoll** [comment]
:+1: I suppose with 127.0.0.1 set it will automatically work locally as it did before?

**Comment by witlessbird** [comment]
@mmoll: correct. In fact that the default value for :dhcp_server parameter when it's not set in the config file.

**Comment by mmoll** [comment]
please rebase

**Comment by witlessbird** [comment]
@mmoll: rebased!

**Comment by mmoll** [comment]
tests fail (just the line order)

**Comment by witlessbird** [comment]
@mmoll: fixed the test.

**Comment by mmoll** [comment]
Спасибо Дмитрий!

**Pull request merged**